perfmetrics/scripts/testing_on_gke/examples/dlio/parse_logs.py

```diff
@@ -185,7 +193,26 @@ def fetch_cpu_memory_data():
                   r["end"],
                   project_number=args.project_number,
               )
-            pass
+            else:
+              r["lowest_memory"], r["highest_memory"] = (
+                  get_memory_from_monitoring_api(
+                      pod_name=r["pod_name"],
+                      start_epoch=r["start_epoch"],
+                      end_epoch=r["end_epoch"],
+                      project_id=args.project_id,
+                      cluster_name=args.cluster_name,
+                      namespace_name=args.namespace_name,
+                  )
+              )
+              r["lowest_cpu"], r["highest_cpu"] = get_cpu_from_monitoring_api(
+                  pod_name=r["pod_name"],
+                  start_epoch=r["start_epoch"],
+                  end_epoch=r["end_epoch"],
+                  project_id=args.project_id,
+                  cluster_name=args.cluster_name,
+                  namespace_name=args.namespace_name,
+              )
+          pass
```

Comment:
pass is only required if the class definition/method block is empty. Here it's a no-op.

Reply:
done. removed all useless `pass` statements from everywhere.